README.md

```diff
@@ -46,3 +46,71 @@ to acknowledge contributors and describe the publication record that is created
 by archiving it using [zenodo.org](https://zenodo.org/). You may also want to
 consider acknowledging contributors with the
 [allcontributors bot](https://allcontributors.org/docs/en/bot/overview).
+
+
+## Dataverse docker
+
+The [dataverse-docker](https://github.com/IQSS/dataverse-docker) repository 
+provides everything needed to run dataverse in a container. The continuous 
+integration test build is based on that. The setup script used for the 
+respective AppVeyor build is under `tools/ci/setup_docker_dataverse`.
+
+You can use this docker setup locally on your machine, too, provided it's 
+running Linux and you have `docker` and `docker-compose` installed. All 
+involved scripts (from our end as well as the dataverse-docker repo) are 
+completely ignorant of Windows.
+The basic setup is this:
+
+    git clone https://github.com/IQSS/dataverse-docker
+    cd dataverse-docker
```

Comment:
As mentioned below, it's necessary to find the docker-compose and the `.env` file. Alternative (as also mentioned) would be to pass those paths to additional options. Official instructions simply lack that hint. All paths given to env vars etc. rely on `.` being that directory or need to be adjusted accordingly.

Reply:
Just an observation: I played with this setup, and I observed a weird failure to create the dataverse when I did this in my home directory (other directories were fine, so I guess Appveyor isn't affected because it does its thing in subdirectories). But not using cp and instead running ``docker-compose --file dataverse-docker/docker-compose.yml -p dataverse-docker up -d`` succeeded in that case. Don't know why, but good to know maybe